apps/editing-toolkit/editing-toolkit-plugin/help-center/class-help-center.php

```diff
@@ -127,7 +134,17 @@ public function enqueue_script() {
 	 * Get current site details.
 	 */
 	public function get_current_site() {
-		$site     = \Jetpack_Options::get_option( 'id' );
+		$is_support_site = defined( 'WPCOM_SUPPORT_BLOG_IDS' ) && in_array( get_current_blog_id(), WPCOM_SUPPORT_BLOG_IDS );
+
+		if ( $is_support_site ) {
```

Comment:
In this context, renders the primary user's  on Help Center instead of the support site.
<img width="432" alt="image" src="https://user-images.githubusercontent.com/2653810/221896035-c914ea41-aef9-43c9-8c76-982f44ed88cf.png">